src/linear_algebra/direct_sum_module.lean

```diff
@@ -22,9 +22,11 @@ variables {R ι M}
 instance : semimodule R (direct_sum ι M) := dfinsupp.to_semimodule
 
 variables R ι M
-def lmk : Π s : finset ι, (Π i : (↑s : set ι), M i.1) →ₗ[R] direct_sum ι M :=
+/-- This is like `mk`, but indexed by a finset. -/
```

Comment:
I'm not a fan of docstrings that start with "This is like ...": now I also have to look up `mk`'s docstring to understand this. Can you write something like "Create an element of the `direct_sum` that matches the given values and is `0` otherwise. This is like `mk`, but indexed by a finset." (fixing of course what I misunderstood).

(I'm also not sure what `mk` exactly refers to, when I didn't read through the whole context of this definition.)

Reply:
I had asked @jalex-stark about this file. I had been reading it and I think I now understand it. 
I'll work on fixing this docstring and adding more detailed documentation to this page.